src/python_interpreter/get_interpreter_metadata.py

```diff
@@ -19,6 +19,23 @@
 else:
     ext_suffix = sysconfig.get_config_var("EXT_SUFFIX")
 
+
+def get_abi_tag():
+    # This should probably return the ABI tag based on EXT_SUFFIX in the same
+    # way as pypa/packaging. See https://github.com/pypa/packaging/pull/607.
+    # For simplicity, we just fix it up for GraalPy for now and leave the logic
+    # for the other interpreters untouched, but this should be fixed properly
+    # down the road.
```

Comment:
i'd be happy to switch this over to what pypa/packaging does entirely, their logic is authoritative for all other tools anyway

Reply:
It's tracked here: https://github.com/PyO3/maturin/issues/1545